app/views/devise/two_factor_authentication/show.html.slim

```diff
@@ -11,7 +11,7 @@ p
 
 = form_tag([:user, :two_factor_authentication], method: :put, role: 'form') do
   .mb2
-    = label_tag 'code', raw('<abbr title="required">*</abbr> ') + t('forms.two_factor.code')
```

Comment:
`<abbr title="required">*</abbr>`
This was required for 508-compliance and will likely need to return. I think switching to a simple_form helper will make this cleaner.